batch_uploads_tarchive

```diff
@@ -9,9 +9,29 @@ if(-f "$ENV{LORIS_CONFIG}/.loris_mri/prod") {
     { package Settings; do "$ENV{LORIS_CONFIG}/.loris_mri/prod" }
 } ##Possibly the script can exit if the prod doesn't exist
 #######################################################################################
+
+################################################################
+######### Establish database connection ########################
+################################################################
+my $dbh = &NeuroDB::DBI::connect_to_db(@Settings::db);
+print "\nSuccessfully connected to database \n";
+
 # define project space
 my ($debug, $verbose) = (0,0);
-my ($stdoutbase, $stderrbase) = ("$Settings::data_dir/batch_output/tarstdout.log", "$Settings::data_dir/batch_output/tarstderr.log");
+my $data_dir = &NeuroDB::DBI::getConfigSetting(
+                    $dbh,'dataDirBasepath'
+                    );
+my $tarchiveLibraryir = &NeuroDB::DBI::getConfigSetting(
```

Comment:
here, it should spell $tarchiveLibraryDir (missing the D) making it crash in line 70